sympy/solvers/ode/tests/test_systems.py

```diff
@@ -1071,8 +1071,8 @@ def test_sysode_linear_neq_order1_type2():
     eqs6 = [Eq(Derivative(f(x), x), -9*f(x) - 4*g(x)),
             Eq(Derivative(g(x), x), -4*g(x)),
             Eq(Derivative(h(x), x), h(x) + exp(x))]
-    sol6 = [Eq(f(x), C1*exp(-4*x)*Rational(-4, 5) + C2*exp(-9*x)),
-            Eq(g(x), C1*exp(-4*x)),
+    sol6 = [Eq(f(x), C2*exp(-4*x)*Rational(-4, 5) + C1*exp(-9*x)),
+            Eq(g(x), C2*exp(-4*x)),
```

Comment:
I'm still unclear why this has changed. I don't see a reason that the changes here should change in any way the output of `linsolve` or `linear_eq_to_matrix`.

Reply:
unresolved terms that are zero get numbered; they are no longer retained

Reply:
You can see an example of an expression with `Integrate(0, t)` in the changed tests.

Reply:
Okay, I see.